rpm2cpio/plan.sh

```diff
@@ -0,0 +1,30 @@
+pkg_origin=origin
+pkg_name=rpm2cpio
+pkg_version='1.3'
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=('MIT')
+pkg_description=('Convert .rpm files for extraction with cpio')
+pkg_source='http://svnweb.freebsd.org/ports/head/archivers/rpm2cpio/files/rpm2cpio?revision=259745'
+pkg_shasum=09651201a34771774fc4feaf5b409717e4bc052b82a89f3fc17c0cf578f8e608
+pkg_filename=${pkg_name}
+pkg_deps=(core/perl)
+pkg_build_deps=()
+pkg_bin_dirs=(bin)
+
+pkg_upstream_url='http://svnweb.freebsd.org/ports/head/archivers/rpm2cpio/pkg-descr?revision=259745&view=markup'
+
+do_unpack() {
+  return 0
+}
+
+do_build() {
+  mv "$HAB_CACHE_SRC_PATH"/"$pkg_filename" "$HAB_CACHE_SRC_PATH"/"$pkg_name"-"$pkg_version"/"$pkg_filename"
```

Comment:
this might be my own ignorance in how the folders are being laid out.  All I need to do is just download it and move it into `$pkg_prefix/bin`.   So it sounds like I can just do:

```bash
do_unpack() {
  return 0
}

do_build() {
  return 0
}

do_install() {
  install 0655 "$HAB_CACHE_SRC_PATH"/"$pkg_name" "$pkg_prefix"/bin/"$pkg_name"
}
```